pkg/securityscan/scanHandler.go

```diff
@@ -211,9 +212,48 @@ func (c *Controller) getDefaultClusterScanProfile(clusterprovider string) (strin
 	if !ok {
 		profileName = cm.Data["default"]
 	}
+	lines := c.splitLines(profileName)
+	if len(lines) > 1 {
+		logrus.Debugf("profilename is splitted by k8s version %v", lines)
+		for _, line := range lines {
+			parts := strings.Split(line, ":")
+			if len(parts) > 1 {
+				k8sRange := parts[0]
+				profile := parts[1]
+				// validate cluster's k8s version matches the profile's k8s version range
+				clusterK8sToMatch, err := semver.Make(clusterK8sVersion[1:])
+				if err != nil {
+					logrus.Errorf("Cluster's k8sVersion is not sem-ver %s %v", c.KubernetesVersion, err)
+					continue
+				}
+				if k8sRange != "" {
+					benchmarkK8sRange, err := semver.ParseRange(k8sRange)
+					if err != nil {
+						logrus.Errorf("K8s Range set for profile %s not proper sem-ver %v, error: %v", profile, k8sRange, err)
+						continue
+					}
+					if !benchmarkK8sRange(clusterK8sToMatch) {
+						logrus.Errorf("Kubernetes version mismatch, ClusterScanProfile %v is not valid for this cluster's K8s version %v", profile, c.KubernetesVersion)
```

Comment:
This should not be an error, it's expected that some version won't match, this should probably live at debug level

Reply:
done